### DISCUSSION TITLE
GeoPDF in WebGIS 5/6 ??

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#165] [Ideas]
Wäre inzwischen eine Unterstützung von GeoPDF in WebGIS5/6 möglich? 
Jürgen hatte diese Frage vor ein paar Jahren schon mal gestellt, damals war es leider nicht möglich. Nachdem sich aber evtl. die technischen Möglichkeiten geändert haben, wollen wir nochmal einen Vorstoss wagen.

Unser Vorstellung:
Die Unterstützung bzw. Erstellung eines GeoPDF in WebGIS5/6, d.h. ein PDF das
-	Georeferenziert ist
-	Vektor- und Rasterinformationen in unterschiedlichen Ebenen darstellt (welche schaltbar sind)
-	Suchen und Messen ermöglicht

**Comment by gis-eni** [comment]
WebGIS bezieht seine Daten von Kartenservern, sprich, es werden "nur" Bilder abgeholt und keine Vektordaten. 
Darum bringen GeoPDF eher nix. 

Man könnte zwar **theoretisch** auch Vektordaten über dein AGS/gView Abfragen. Allerdings müsste das WebGIS dann auch noch wissen, wie man diese Darstellen muss (Farben, etc) was meiner Meinung nach einen riesen aufwand in der Programmierung und vor allem auch in der Parametrierung bedeuten würde.

Daher klares NEIN.

WebGIS als Produkt zum Erzeugen von GeoPDFs wäre ja auch der falsche Zugang. Wenn man so komplexe PDFs haben möchte, braucht es meiner Meinung nach ein eigenes Service, das so etwas erstellt (mit direkten Zugriff auf die Daten, am besten auch auf die Darstellung). So etwas müsste man entweder gleich in den Kartenserver implementieren (also neben JPG/PNG Export auch noch PDF Export) oder man baut eine komplett eigene Anwendung, die so ein PDF erzeugt.

Wenn so eine Anwendung existiert, bin ich gerne bereit, das WebGIS so umzubauen, dass beim Drucken, das PDF von einer Drittanwendung abgeholt wird.

**Comment by gis-eni** [comment]
Eventuell Spezialkarten über Scripts erstellen (GDAL) und als ProcServer Job anbieten
